lib/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfing_viewmodel.dart

```diff
@@ -356,6 +359,41 @@ class WorkGroupNodesSurfingViewModel extends BaseViewModel {
     };
   }
 
+  OnlineThunkAction downloadNodeAction(List<WorkGroupNode> nodes, {ItemSelectionType itemSelectionType = ItemSelectionType.single}) {
```

Comment:
if you return an `OnlineThunkAction` then you have to call `store.dispatch(yourAction)` to execute it. But in this case this should be a normal `void` function `void downloadNodeAction(...)` to be called from widget